assets/scripts/src/choices.js

```diff
@@ -2348,6 +2347,14 @@ class Choices {
       dropdown.insertBefore(input, dropdown.firstChild);
     }
 
+    if (
+      this.placeholder &&
+      this.passedElement.type !== 'select-one' &&
+      !this.config.searchPlaceholderValue
+    ) {
+      input.style.width = getWidthOfInput(input);
```

Comment:
Yes, me neither in this project. But in my larger project it won't work without it. But I am ok to leave as it is now without async call and do it myself in `callbackOnInit`.